IdentityCore/src/MSIDError.m

```diff
@@ -49,3 +49,20 @@
 }
 
 
+MSIDErrorCode MSIDErrorCodeForOAuthError(NSString *oauthError, MSIDErrorCode defaultCode)
+{
+    if (oauthError && [oauthError caseInsensitiveCompare:@"invalid_request"] == NSOrderedSame)
```

Comment:
We already have this code in token response I think. Can we combine it?

Reply:
Where in token response do we have this?

Reply:
done, i've changed TokenResponse to call this.